test/fixtures_redis_test.go

```diff
@@ -54,12 +54,10 @@ func TestFixturesRedisSuite(t *testing.T) {
 
 func (s FixturesRedisSuite) TestRedis() {
 	config := cfg.New()
-	config.Option(
+	_ = config.Option(
```

Comment:
how dare you to ignore the error :P

Reply:
In the `testing` universe, everything is allowed ;)

Reply:
i guess it's going to fail anyway if something would have errored